### ISSUE TITLE
LiT17 dataset source

### ISSUE THREAD

**Issue by cheliu-computation** [med-air/3DSAM-adapter#1]
Hi, authors, thanks for your impressive work on 3D SAM for medical image analysis.

I am downloading the LiT17 dataset from the original link provided in this repo, however the competition seems over and not allow for further downloading. Also I checked the processed data from your link. 
The processed data looks fine on /train folder, every volume has a seg file. But in /folder, there is only one file for each volume, from your dataset code, it seems the segment mask is combined with the raw volume,  

I am not sure I understand correct, because the data format seems quite different with the original LiT17. Because I find the LiT17 format in this https://github.com/RL4M/PCRLv2/tree/finetune , and it shows every volume should pair with one seg file in train/test folder.

Looking forward to your reply!

**Comment by peterant330** [comment]
Hi,

The LiTS 2017 released 131 volumes with labels for training and validation. For the test set, only the raw image is released. As a result, we did not use the test data in our paper. We split the original training data into our training/validation/test.

Actually, you can still download the data from the official website https://competitions.codalab.org/competitions/17094#participate-get-data, which only requires registration.

Best

> Hi, authors, thanks for your impressive work on 3D SAM for medical image analysis.
> 
> I am downloading the LiT17 dataset from the original link provided in this repo, however the competition seems over and not allow for further downloading. Also I checked the processed data from your link. The processed data looks fine on /train folder, every volume has a seg file. But in /folder, there is only one file for each volume, from your dataset code, it seems the segment mask is combined with the raw volume,
> 
> I am not sure I understand correct, because the data format seems quite different with the original LiT17. Because I find the LiT17 format in this https://github.com/RL4M/PCRLv2/tree/finetune , and it shows every volume should pair with one seg file in train/test folder.
> 
> Looking forward to your reply!

**Comment by csp2002** [comment]
Hi, I noticed that the dataset LiTS 2017 includes 131 volumes, but your split.pkl only includes 118 volumes. Is there any reason for excluding 13 volumes?
